health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/HypothesisWrapper.js

```
@@ -313,6 +314,43 @@ const HypothesisWrapper = ({ onSelect, props: customProps }) => {
+    // console.log(assumptionValues);
+    const currentAssumptions = assumptionValues?.filter(assumption => assumption?.category === currentCategory);
+
+    //if we go back with some invalid value show a toast to correct it
+    function hasInvalidValues(currentAssumptions) {
+      return currentAssumptions
+        .filter((item) => item?.value !== "") // Filter out empty string values
+        .some((item) => {
+          const value = Number(item?.value); // Parse the value into a number
+    
+          // Check if the value is invalid
+          if (isNaN(value) || value < 0 || value > 1000) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)